tools/wptserve/tests/functional/base.py

```diff
@@ -36,16 +35,16 @@ def add_data(self, data):
         BaseRequest.add_data(self, data)
 
 
-class TestUsingServer(unittest.TestCase):
-    def setUp(self):
+class TestUsingServer:
```

Comment:
Let's create a `conftest.py` file and use pytest fixtures rather than this kind of class-based setup.